### PR TITLE
improve cover video spacing and sizing

### LINKED ISSUE #1710: Cover featured video improvements
- Add a left to center gradient to improve readability of text
- Make the gradient fixed and placed at the bottom - it doesn't need to resize that much, just keep its container aligned with the video carousel
- Improve the positioning of the info container so it's fixed w.r.t. to the first video carousel
- the subtext:
  - increase the margin
  - lower the max font size
  - set fixed max width instead of % based

### PR DESCRIPTION
**Pull request by kdembler**
### Review #1749 first

Resolves #1710 
- tries to make the gradient sizing more logical
- fix the info container w.r.t. the first video carousel
- add a horizontal gradient
- improve font spacing and sizing 

Screenshot 2020-11-18 at 16 39 17

### CODE REVIEW

File: atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts [hunk 1]
```diff
@@ -131,19 +176,22 @@ export const TitleContainer = styled.div`
   }
 
   h2 {
-    ${fluidRange({ prop: 'fontSize', fromSize: '40px', toSize: '60px' })};
-    ${fluidRange({ prop: 'lineHeight', fromSize: '48px', toSize: '68px' })};
+    ${fluidRange({ prop: 'fontSize', fromSize: '40px', toSize: '72px' }, '320px', '1920px')};
```
Comment (fcescob) [inline atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts]: If possible, let's use a breakpoint from the theme here

Comment (kdembler) [reply atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts]: ah yes, good catch

Comment (kdembler) [reply atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts]: Fixed

**Review by fcescob — APPROVED** [review]
LGTM!

**Pull request merged**